frontend/src/static/js/v_summary.js

```diff
@@ -246,6 +255,10 @@ window.vSummary = {
       if (hash.breakdown) {
         this.filterBreakdown = convertBool(hash.breakdown);
       }
+      if (hash.checkedFileTypes) {
+        const parsedFileTypes = hash.checkedFileTypes.split(window.HASH_FILETYPE_DELIMITER);
+        this.checkedFileTypes = parsedFileTypes.filter((type) => this.fileTypes.includes(type));
+      }
```

Comment:
is there really a need to filter?

Reply:
yes, there may be a case where the hash contains unnecessary noise, so this filters it out.